config/locales/en.yml

```diff
@@ -811,6 +811,8 @@ en:
       remember_me: "Remember me"
       reset_by_clicking_here: "Forgot your password? <a href='%{url1}'>Reset it here</a>"
   home:
+    read_more: "Read more"
+    more_projects: "More projects"
```

Comment:
Hi @KarishmaVanwari, there should be a home key before the read_more key. You can take a look at the issue again to see what I’m talking about.

Also the rest of the files added to this PR are not a part of the changes required. To remove them, all you have to do is undo the changes you made to them and then do a push again. Thank you!